File: src/main/kotlin/cc/stevenyin/actioninkotlin/chapter02/_02_03/Person.java
package cc.stevenyin.actioninkotlin.chapter02._02_03;

public class Person {
	private final String name;
	public Person(String name) {
		this.name = name;
	}
//  final 成员变量, 没有setter方法
	public String getName() {
		return name;
	}
}

File: src/main/kotlin/com/stevenyin/chapter02/bean/Person.kt
package com.stevenyin.bean

//public class Person {
//	private final String name;
//	private final int age;
//  private Boolean isMarried;
//	@getter
//  final 成员变量, 没有setter方法
//  public void setIsMarried(Boolean isMarried) {
//      this.isMarried = isMarried;
//  }
//}

class Person (val name: String, val age: Int, var isMarried: Boolean)

fun main() {
	val person = Person("Jack", 20, true)
	println(person)
}